### PR TITLE
QA-365: Add the conventional commit spec to the PR template

### PR DESCRIPTION
**Pull request by oleorhagen**
Changelog: None
Signed-off-by: Ole Petter <ole.orhagen@northern.tech>

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 <!-- AUTOVERSION: "/mender/blob/%"/ignore -->
 🚨 Please review the [guidelines for contributing](https://github.com/mendersoftware/mender/blob/master/CONTRIBUTING.md) to this repository.
 
+- [ ] Make sure that all commits follow the conventional commit [specification](https://www.github.com/mendersoftware/mendertesting/commitlint/grammar.md) for the Mender project.
```
Comment (kacf) [inline .github/PULL_REQUEST_TEMPLATE.md:7]: To lower the entry barrier, would it be helpful with an example here? I think the vast majority of contributions will fall into this category, don't you agree?

```
fix: <SHORT DESCRIPTION OF FIX>

<OPTIONAL LONGER DESCRIPTION>

Changelog: <USER-FRIENDLY-CHANGE-DESCRIPTION>
Ticket: <TICKET NUMBER OR "None">
```

Comment (oleorhagen) [reply .github/PULL_REQUEST_TEMPLATE.md:7]: Good idea!  Adding :)

Comment (oleorhagen) [reply .github/PULL_REQUEST_TEMPLATE.md:7]: https://github.com/mendersoftware/mender-docs/commit/5ada39407b2b1fe7ddbebcc4e1d13e90100e85e5

@kacf what do you think about this?

Comment (kacf) [reply .github/PULL_REQUEST_TEMPLATE.md:7]: Perfect!

**Review by kacf — APPROVED** [review]

**Comment by kacf** [comment]
I assume you'll update the other PRs too then?

**Pull request merged**